Godeps/Godeps.json

```diff
@@ -0,0 +1,9 @@
+{
+	"ImportPath": "github.com/lilic/mountinfo",
```

Comment:
aha yes it was autogenerated `github.com/kinvolk/mountinfo-graph` this would be the correct one then?